docs/02_database/01_fields-tables-views/02_tables/02_tables-advanced.md

```diff
@@ -66,120 +66,89 @@ table(name = "GENESIS_PROCESS", id = 12) {
-        ...
+The audit table that is created has the same name as the source table, plus the suffix **_AUDIT**. Here is an example table definition, which creates a table called `EUR_TRADES` and an audit table called `EUR_TRADES_AUDIT`:
 
-```
-
-#### Example
-
-```kotlin
-table (name = "TRADE", id = 2000, audit = details(id = 2100, sequence = "TR_AU", tskey = true)) {
+```kotlin {1}
+table (name = "EUR_TRADE", id = 2000, audit = details(id = 2100, sequence = "TR")) {
     sequence(TRADE_ID, "TR")
     COUNTERPARTY_ID 
     INSTRUMENT_ID not null
     QUANTITY
     PRICE not null
-    SYMBOL
-    DIRECTION
     TRADE_DATE
-    ENTERED_BY
     TRADE_STATUS
 
     primaryKey {
         TRADE_ID
     }
 }
-
 ```
 
-### Event Handlers with auditing tables
+Whenever an `eventHandler` interacts with the table `EUR_TRADES`, a record is automatically inserted to the audit table `EUR_TRADES_AUDIT`to log the event.
 
-In order to be able insert automatically audit records to audit tables, you only need to follow the [previous steps](#adding-audit-to-table-dictionary). After that, every time an eventhandle interacts with any audited table, it will automatically insert an audit record.
+So, to be clear:
 
-To garantee that the audit record is inserted into the audit table, Genesis provide a parameter to eventhandlers called `transactional = true | false`.
+- An audit table monitors and logs details of changes on a specific table. 
+- If the table and the `eventHandlers` that interact with it are set up correctly, then the auditing is performed automatically. 
+- Every change to the table is automatically logged and timestamped on the audit table.
 
-**Transactional** eventhandlers are [ACID](../../../../getting-started/glossary/glossary/#acid)-compliant, which means that it will use the concept of transactions to garantee that the audit record is inserted into the audit table. In other words, if `transactional = true` and the eventhandler triggered finishes its excecution, then it is garanteed that the audit record is inserted.
+### Structure of audit tables
 
-:::warning
-Make sure your database supports transactions
-:::
+When you create an audit table, it has all the same fields as the source table, plus the following fields:
 
+| Field name                | Data Type | Description |
+| ---                       | ---       | ---  |
+| `AUDIT_EVENT_DATETIME`    | DATATIME  | Autogenerated date and time of the event  |
+| `AUDIT_EVENT_TEXT`        | STRING    | Optional “REASON” value sent as part of the event message | 
+| `AUDIT_EVENT_TYPE`        | STRING    | Event that wrote on the source table |
+| `AUDIT_EVENT_USER`        | STRING    | User on the event message |
+ 
+### ACID
+Every `eventHandler` that interacts with the audited table must be [transactional](../../../../server/event-handler/basics/#transactional-event-handlers-acid). This ensures [ACID](../../../../getting-started/glossary/glossary/#acid) compliance.
```

Comment:
I may have not understood, eventhandlers can interact with audited tables even if they are not transactional. It is always a good practice to keep them transactional, but in fact you can do it. Isn't it the case to change "must be transactional" to "should be transactional"?